### ISSUE TITLE
Developer's Tools desklet - Styling on Cinnamon is hard to read (light green text on light gray)

### ISSUE THREAD

**Issue by TopView** [linuxmint/cinnamon-spices-desklets#168]
Have used and loved this thing for quite some time on Debian Jessie w/ Cinnamon, but when I upgraded today the stying turned funky on me.  

My current desktop theme: BlueMenta  (or any of the blue or green background themes below).  

Update: If I choose Cinnamon theme this is a workaround and it displays black on white.  

The blue and green themes seem to cause this issue:

![screenshot-area-2017-06-29-234534](https://user-images.githubusercontent.com/2180096/27724157-5d63d1f2-5d25-11e7-8c1a-866205f9c969.png)

See this snap:  
  ![screenshot-cinnamon-2017-06-29-232607](https://user-images.githubusercontent.com/2180096/27723603-c9b6f904-5d22-11e7-8e87-fd19c2d82e35.png)

I think the issue is the background or something like that.  Thanks!

Debian 9.0 stretch / Cinnamon version 3.2.7

![screenshot-area-2017-06-29-233146](https://user-images.githubusercontent.com/2180096/27723692-2a32f81e-5d23-11e7-902c-a12232687b47.png)

cinnamon log:

info:  6/29 23:08:35  Cinnamon.AppSystem.get_default() started in 184 ms
info:  6/29 23:08:35  loading user theme: /usr/share/themes/BlueMenta/cinnamon/cinnamon.css
info:  6/29 23:08:35  added icon directory: /usr/share/themes/BlueMenta/cinnamon
info:  6/29 23:08:35  PlacesManager: Updating devices
info:  6/29 23:08:35  loaded at Thu Jun 29 2017 23:08:35 GMT-0700 (PDT)
info:  6/29 23:08:36  Role locked: panellauncher
info:  6/29 23:08:36  Loaded applet panel-launchers@cinnamon.org in 301 ms
info:  6/29 23:08:36  Loaded applet window-list@cinnamon.org in 46 ms
info:  6/29 23:08:38  Loaded applet menu@cinnamon.org in 2182 ms
info:  6/29 23:08:38  Loaded applet systray@cinnamon.org in 9 ms
info:  6/29 23:08:38  Loaded applet separator@cinnamon.org in 7 ms
info:  6/29 23:08:38  Loaded applet show-desktop@cinnamon.org in 13 ms
info:  6/29 23:08:38  Loaded applet recent@cinnamon.org in 24 ms
info:  6/29 23:08:38  Loaded applet windows-quick-list@cinnamon.org in 7 ms
info:  6/29 23:08:38  Role locked: notifications
info:  6/29 23:08:38  Loaded applet notifications@cinnamon.org in 17 ms
info:  6/29 23:08:38  Loaded applet sound@cinnamon.org in 67 ms
info:  6/29 23:08:38  Loaded applet network@cinnamon.org in 81 ms
info:  6/29 23:08:38  Loaded applet calendar@cinnamon.org in 23 ms
info:  6/29 23:08:38  Loaded applet power@cinnamon.org in 30 ms
info:  6/29 23:08:38  undefined: Spawned process with pid=17197
info:  6/29 23:08:38  capture@rjanja: Using AppletSettings
info:  6/29 23:08:38  capture@rjanja: [disabled] imgur
error:  6/29 23:08:38  Could not find setting key 'kb-cs-monitor-1' for applet/desklet uuid capture@rjanja
info:  6/29 23:08:38  Loaded applet capture@rjanja in 263 ms
info:  6/29 23:08:38  Loaded applet graphicsCenter@scollins in 116 ms
info:  6/29 23:08:39  Loaded applet weather@mockturtl in 56 ms
info:  6/29 23:08:39  Loaded applet expo@cinnamon.org in 28 ms
info:  6/29 23:08:39  AppletManager.init() started in 3302 ms
info:  6/29 23:08:39  Loaded desklet launcher@cinnamon.org in 37 ms
info:  6/29 23:08:39  Loaded desklet clock@cinnamon.org in 22 ms
info:  6/29 23:08:39  Loaded desklet photoframe@cinnamon.org in 42 ms
info:  6/29 23:08:39  Loaded desklet devTools@scollins in 67 ms
info:  6/29 23:08:39  Cinnamon took 3854 ms to start
info:  6/29 23:08:40  capture@rjanja: Support for open folder/file enabled
info:  6/29 23:08:41  Hiding systray: network
info:  6/29 23:08:51  loading user theme: /usr/share/themes/BlueMenta/cinnamon/cinnamon.css
info:  6/29 23:08:51  added icon directory: /usr/share/themes/BlueMenta/cinnamon
info:  6/29 23:08:55  loading user theme: /usr/share/themes/BlueMenta/cinnamon/cinnamon.css
info:  6/29 23:08:55  added icon directory: /usr/share/themes/BlueMenta/cinnamon
info:  6/29 23:09:41  loading user theme: /usr/share/themes/BlueMenta/cinnamon/cinnamon.css
info:  6/29 23:09:41  added icon directory: /usr/share/themes/BlueMenta/cinnamon
info:  6/29 23:10:03  loading user theme: /usr/share/themes/BlueMenta/cinnamon/cinnamon.css
info:  6/29 23:10:03  added icon directory: /usr/share/themes/BlueMenta/cinnamon
info:  6/29 23:10:07  loading user theme: /usr/share/themes/BlueMenta/cinnamon/cinnamon.css
info:  6/29 23:10:07  added icon directory: /usr/share/themes/BlueMenta/cinnamon
info:  6/29 23:10:08  loading user theme: /usr/share/themes/BlueMenta/cinnamon/cinnamon.css
info:  6/29 23:10:08  added icon directory: /usr/share/themes/BlueMenta/cinnamon
info:  6/29 23:10:13  loading user theme: /usr/share/themes/BlueMenta/cinnamon/cinnamon.css
info:  6/29 23:10:13  added icon directory: /usr/share/themes/BlueMenta/cinnamon
info:  6/29 23:10:14  loading user theme: /usr/share/themes/BlueMenta/cinnamon/cinnamon.css
info:  6/29 23:10:14  added icon directory: /usr/share/themes/BlueMenta/cinnamon
info:  6/29 23:13:06  Loaded desklet calculator@scollins in 58 ms
info:  6/29 23:13:11  22 desklet already loaded
info:  6/29 23:13:53  Installing settings for bbcwx@oak-wood.co.uk
info:  6/29 23:13:53  Settings successfully installed for bbcwx@oak-wood.co.uk
info:  6/29 23:13:53  bbcwx (instance 23): refreshing forecast at 23:13:53
info:  6/29 23:13:53  Loaded desklet bbcwx@oak-wood.co.uk in 66 ms
info:  6/29 23:14:00  23 desklet already loaded
info:  6/29 23:15:31  Installing settings for soundBox@scollins
info:  6/29 23:15:31  Settings successfully installed for soundBox@scollins
info:  6/29 23:15:31  Hiding systray: network
info:  6/29 23:15:31  Hiding systray: network
info:  6/29 23:15:31  Hiding systray: network
info:  6/29 23:15:31  Hiding systray: network
info:  6/29 23:15:31  Hiding systray: network
info:  6/29 23:15:31  Hiding systray: network
info:  6/29 23:15:31  Hiding systray: network
info:  6/29 23:15:31  Hiding systray: network
info:  6/29 23:15:31  Hiding systray: network
info:  6/29 23:15:31  Hiding systray: network
info:  6/29 23:15:31  Hiding systray: network
info:  6/29 23:15:31  Hiding systray: network
info:  6/29 23:15:31  Hiding systray: network
info:  6/29 23:15:31  Hiding systray: network
info:  6/29 23:15:31  Hiding systray: network
info:  6/29 23:15:31  Hiding systray: network
info:  6/29 23:15:31  Loaded desklet soundBox@scollins in 139 ms
info:  6/29 23:15:36  24 desklet already loaded
info:  6/29 23:16:33  Hiding systray: network
info:  6/29 23:17:41  21 desklet already loaded
info:  6/29 23:25:46  Initializing screenshot tool
info:  6/29 23:26:07  Initializing screenshot tool
info:  6/29 23:31:46  Initializing screenshot tool
info:  6/29 23:31:54  Selection area is 492,232 - 938 x 232

**Comment by NikoKrause** [comment]
Which desklet are you talking about? Add it in the title.
And which theme do you use?

**Comment by NikoKrause** [comment]
Aren't those MATE themes? Do they work properly on Cinnamon?
I'd say this is a theme issue and not a desklet issue. You should bug report against the themes.

**Comment by collinss** [comment]
@NikoKrause afaik, it's not possible to use a MATE theme in Cinnamon. That being said, it is a theme issue. Devtools uses it's own styling to make sure things are readable no matter what theme is being used, so it shouldn't be getting any styling info from the desktop theme. It looks like the theme in question is overriding something in the desklet style classes, which it shouldn't be.
I went ahead and downloaded the BlueMenta cinnamon theme. I couldn't specifically figure out what in the theme is causing it, but I noticed several other issues with that theme as well. I would personally recommend that you use a desktop theme that has a better support for current versions of Cinnamon, or else contact the author, and request that it be fixed in the theme. Unfortunately, there really isn't anything I can do to prevent theme developers from making bad decisions, so I'm going to close this issue.
